.github/workflows/build-cli.yml

```diff
@@ -35,11 +35,13 @@ jobs:
             target-suffix: unknown-linux-gnu
             build-on: ubuntu-latest
             use-cross: true
+            cc: gcc-10
```

Comment:
looks like latest stable version of gcc is 15. could we update all the way to newest?

Reply:
quite possibly, yeah. I went with 10 because ubuntu 20 (what we're building on) ships with 9, and 10 is where the original issue is fixed: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=95189

according to some googling though, 10 is the latest in the official repositories for ubuntu 20 (I haven't verified) so it might mean some extra steps to get it working for not much gain